cves/CVE-2016-3092.yml

```diff
@@ -43,37 +43,32 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+  The multipart stream area of the system doesn't check the length of a boundary string. This
+  means that a user can potentially use an extremely long boundary string to cause a denial of
+  service attack.
 
 bounty_instructions: |
   If you came across any indications that a bounty was paid out for this
   vulnerability, fill it out here. Or correct it if the information already here
   was wrong. Otherwise, leave it blank.
 bounty:
   amt:
-  announced: 2016-07-04 
+  announced: 
   url:
-bugs: []
+bugs: [91453]
 fixes_vcc_instructions: |
   Please put the Git commit SHA in "commit" below, and any notes about how this
   was discovered in the "note" field.
   Refer to our instructions on how to find a Git SHA from an SVN revision.
 fixes:
    - commit: 8e67e4a4799a9765b3bd777eed3ee17f00514441
-     note: SVN rev 1743700, from the Tomcat website.
-   - commit: 2c3553f3681baf775c50bb0b49ea61cb44ea914f
-     note: SVN rev 1743722, from the Tomcat website.
-   - commit: d752a415a875e888d8c8d0988dfbde95c2c6fb1d
-     note: SVN rev 1743738, from the Tomcat website.
-   - commit: 8999f8243197a5f8297d0cb1a0d86ed175678a77
-     note: SVN rev 1743742, from the Tomcat website.
-   - commit:
-     note:
+     note: |
+       Prevents the user from making a boundary string that is just shorter than the
```

Comment:
Maybe add the commit where the "too long" boundary string case is fixed

Reply:
Sounds good. I'll add that.